input/fsh/profiles/AsPractitionerRoleProfile.fsh

```diff
@@ -0,0 +1,210 @@
+Profile: 		ASPractitionerRoleProfile
+Parent: 		PractitionerRole
```

Comment:
A voir avec Isabelle : de quel profil d'IS hériter ?

Reply:
j'ai hérité de FrPractitionerRoleExercice, 
à confirmer (ou pas) par Isabelle